src/app/header/header.component.html

```diff
@@ -1,74 +1,157 @@
-<nav class="navbar navbar-default navbar-pf" role="navigation" *ngIf="loggedIn">
-    <div class="navbar-header"></div>
+<nav class="navbar navbar-inverse navbar-pf" role="navigation" *ngIf="loggedIn">
+  <div class="navbar-header">
+    <ul class="nav navbar-nav navbar-right hidden-sm hidden-md hidden-lg ">
+      <li *ngIf="!loggedIn" >
+        <a (click)='login();'>Sign In</a>
+      </li>
+      <li *ngIf="loggedIn" dropdown class="pull-right dropdown">
+        <a dropdown-open class="pull-right" >
+          <div *ngIf="loggedInUser" id="header_dropdownToggle_header">
+            <span class="nav-item-icon">
+              <span *ngIf="!imgLoaded" class="pficon pficon-user"></span>
+              <img (load)="onImgLoad()" src="{{ loggedInUser.attributes?.imageURL }}" [ngClass]="{'show' : imgLoaded,'img-icon' : imgLoaded, 'hide' : !imgLoaded}" />
+            </span>
+            <span class="nav-item-text" >{{loggedInUser.attributes?.fullName}}</span>
+            <span class="caret"></span>
+          </div>
+        </a>
+        <ul class="dropdown-menu view-width-100">
+          <li>
+            <a [routerLink]="[urlFeatureToggle + '/pmuir']">
+              <span class="nav-item-text">Profile</span>
+            </a>
+          </li>
+          <li>
+            <a href="https://fabric8.io/guide/getStarted/index.html">
+              <span class="nav-item-text">Help</span>
+            </a>
+          </li>
+          <li class="divider"></li>
+          <li>
+            <a [routerLink]="[urlFeatureToggle + '/pmuir/settings']">
+              <span class="nav-item-text">Settings</span>
+            </a>
+          </li>
+          <li>
+            <a *ngIf="loggedIn" (click)='logout();'>
+              <span class="nav-item-text">Sign out</span>
+            </a>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </div>
     <div class="collapse navbar-collapse navbar-collapse-1">
-        <ul class="nav navbar-nav navbar-primary persistent-secondary">
+        <ul class="nav navbar-nav navbar-primary persistent-secondary navbar-left">
             <!-- This part of the menu is dynamic based on context -->
             <li class="context dropdown" dropdown>
                 <a dropdown-open>
-                    <span>
-                        <span class="{{context.current.type.icon}}"></span> {{context.current.name}}
-                    </span>
-                    <b class="caret"></b>
+                    <div *ngIf="loggedInUser" id="header_dropdownToggle">
```

Comment:
The 'header' menu only displays when in mobile mode ... so its not on the left in desktop mode